tools/validator.py

```diff
@@ -42,7 +42,9 @@ def CheckFile(self, filename):
           result = False
 
     except errors.FormatError as exception:
-      logging.warning(exception.message)
+      logging.warning(
+          u'Unable to validate file: {0:s} with error: {1:s}'.format(
```

Comment:
Ack. Note tjat we use the explicit notation across the rest of the codebase so I'm sticking with keeping it to match that style and not to mix styles.